ViewModel/ExpressCheckoutButton.php

```diff
@@ -76,15 +76,19 @@ public function __construct(
      */
     public function shouldRender(): bool
     {
-        return $this->isExpressCheckoutActive();
+        return $this->isExpressCheckoutDisabled();
```

Comment:
this still doesn't make sense. `ShouldRender` should be when expressCheckout is enabled not when it is disabled. right?!